### PR TITLE
add endpoint that can mark a given chat as read/unread

### PR DESCRIPTION
**Pull request by uncleJim21**
attempting to address #774

### CODE REVIEW

**Comment by uncleJim21** [comment]
@kevkevinpal @Evanfeenstra I took a stab at this and I think I am close but I do not have a great way to test this. Could you recommend a way to set up a test environment so I can prove out this does what I expect? Or if it's more convenient just hop on a call with me while you review so I can check out your setup

**Comment by kevkevinpal** [comment]
@uncleJim21 I pushed a new test can you check to see if this looks good to you, it tests for that `seen` field in the chat

**Comment by tomastiminskas** [comment]
Looks ok to me. If @uncleJim21 agrees we can push to my 2 nodes so Jim can test on Tom Sim. Thanks

File: src/controllers/messages.ts [hunk 1]
```diff
@@ -797,6 +797,72 @@ export const receiveDeleteMessage = async (payload: Payload): Promise<void> => {
...
+    }
+    //mark the chat as unseen
+    const chat: Chat = (await models.Chat.findOne({
+      where: { id: chat_id, tenant },
+    })) as Chat
+    if (chat) {
+      resetNotifyTribeCount(parseInt(chat_id))
+      await chat.update({ seen: false })
+      success(res, {})
+      sendNotification(chat, '', 'badge', owner)
```
Comment (kevkevinpal) [inline src/controllers/messages.ts:849]: why are we sending a push notification here? Weird to send a notification for something we do manually

Comment (uncleJim21) [reply src/controllers/messages.ts]: good point. I went ahead and removed that line.

**Comment by tomastiminskas** [comment]
@kevkevinpal can you confirm if this is deployed to Tom Sim node? 

Once you confirm @uncleJim21 can finish the feature testing there

**Pull request merged**